secretctl

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This is to allow us Mac OS users to flex versions of bash that aren't from 2007....